service/test/controllers/organisationmanagement/OrgTypeControllerTest.java

```diff
@@ -0,0 +1,136 @@
+package controllers.organisationmanagement;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static play.test.Helpers.route;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import controllers.BaseControllerTest;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.responsecode.ResponseCode;
+import play.libs.Json;
+import play.mvc.Http.RequestBuilder;
+import play.mvc.Result;
+import play.test.Helpers;
+
+public class OrgTypeControllerTest extends BaseControllerTest {
+
+  private static String orgTypeName = "org-type-name";
+  private static String id = "id";
+
+  @Test
+  public void testgetOrgTypeList() {
+
+    RequestBuilder req = new RequestBuilder().uri("/v1/org/type/list").method("GET");
+    req.headers(headerMap);
+    Result result = route(req);
```

Comment:
In each controller test use BaseController.performTest. e.g. CourseBatchControllerTest